src/pages/ArchaeologistsPage.tsx

```diff
@@ -11,7 +11,7 @@ export function ArchaeologistsPage() {
       direction="column"
     >
       <Heading size="lg">Archaeologists</Heading>
-      <ArchaeologistList includeDialButton={true} />
+      {/* <ArchaeologistList includeDialButton={true} /> */}
```

Comment:
I'm simply asking because I don't know. Are we throwing this page out? If not then we need to find a way to restore it.

Reply:
yeah we should keep this page for now if possible